src/test/script_tests.cpp

```diff
@@ -895,17 +895,19 @@ BOOST_AUTO_TEST_CASE(script_build)
         }
     }
 
+#ifdef UPDATE_JSON_TESTS
```

Comment:
13767 added empty file descriptor_tests.cpp (yet the changes in descriptor_tests are needed and missing https://github.com/dashpay/dash/blob/develop/src/test/descriptor_tests.cpp#L67)

Reply:
Fixed